py3status/modules/spotify.py

```diff
@@ -27,12 +27,23 @@
     color_paused: Song is stopped or paused, defaults to color_degraded
     color_playing: Song is playing, defaults to color_good
 
+Click Events:
+    You can add some config to your i3status.conf to  create click events
+    For example
+
+    - play_paused = 1 ==> left click : Pause/Play
+    - next_song = 4 ==> scroll up : Next song
+    - previous_song = 5 ==> scroll down : Previous song
+
 i3status.conf example:
 
 ```
 spotify {
     format = "{title} by {artist} -> {time}"
     format_down = "no Spotify"
+    next_song = 4
+    previous_song = 5
+    play_paused = 1
```

Comment:
by convention in modules we use `button_next`, `button_play` etc

Reply:
Oups I didn't see that, my bad, i made the changes

Reply:
(Suggestion): Use `button_pause` instead and default this with `1` too.